### DISCUSSION TITLE
Icon loading in header spin infinity in Bootstrap5 theme

### DISCUSSION THREAD

**Discussion by maiduyduc** [Power-Components/livewire-powergrid#981] [Q&A]
It's my fisrt time i  install livewire-powergrid and using it with Bootstrap5 theme.
I'm install and use with document and i got an error.
Icon loading in top left of table spin infinity when no action happens.
How i can fix it, please help

**Comment by maiduyduc** [comment]
![image](https://github.com/Power-Components/livewire-powergrid/assets/57941632/ac96bcf1-eea1-4d19-a80e-1ed816d9b359)

**Comment by lokutus24** [comment]
hello,

i have a same error with BS5.
Can somehow help me?

**Reply by maiduyduc** [reply]
you can turn off with "withoutLoading()" in setup function
```
use PowerComponents\LivewirePowerGrid\Header 

public function setUp(): array
{
     return [
         Header::make()
             ->withoutLoading(),

         // ...
     ];
}
```

**Reply by lokutus24** [reply]
hello,

yes i use this, but why works it not normally with page load?

**Reply by maiduyduc** [reply]
I didn't understand why it was like that, so I had to turn it off
